src/components/Kv/KvMap.vue

```diff
@@ -12,6 +12,9 @@
 <script>
 import { createIntersectionObserver } from '@/util/observerUtils';
 import testDelayedGlobalLibrary from '@/util/timeoutUtils';
+import countriesBorders from '@/assets/data/components/LenderProfile/countries-borders.json';
+import { getIntervals, mapColors } from '@/components/LenderProfile/LenderMap';
```

Comment:
It looks like these two methods might better exist here (or in a shared util file).